qiita_pet/templates/analysis_selected.html

```diff
@@ -1,8 +1,15 @@
 {% extends sitebase.html %}
 {% block head %}
 <script type="text/javascript">
+var ws = new WebSocket('ws://' + window.location.host + '/analysis/selected/socket/');
+ws.onmessage = function(evt) { if(evt.data != 'true') {ws.close();} };
```

Comment:
The connection is persistent, this is just so the websocket is closed if there is an error server side and the expected response isn't given. Then it closes the connection so the error is shown.